juniper_official/routing/check-te-rsvp-interface-errors.rule

```diff
@@ -201,7 +201,7 @@ healthbot {
              * Anomaly detection logic to detect rsvp-te interface errors.
              */
             trigger rsvp-te-interface-errors {
-                frequency 3offset;
+                frequency 1offset;
```

Comment:
increasing-at-least-by-value should also have 2.5 offset

Reply:
resolved

Reply:
resolved